AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -6852,7 +6863,9 @@ function Move-OnPremSharePermissionsToAzureFileShare
         #Geting the OID of domain user/group using its SID
         try
         {
-            $aadUser = Get-AzureADUser -Filter "OnPremisesSecurityIdentifier eq '$strSID'"
+            # if not connected to MsGraph, will ask user to call Connect-MgGraph in error message, need User.Read.All scope.

```

Comment:
Is there a way to connect to MS Graph with given scopes, but reuse cached credentials if the user is already logged in?

Reply:
If the user is already logged in, they don't need to log in again. And when the prompt popped up at the login, it will ask for the permission needed, like "Read user" permission.

Reply:
I have added request MS graph here, so it will auto pop up if user is not signed in.